### PR TITLE
[DRAFT] Optimise Docker Caching

### PR DESCRIPTION
**Pull request by geektype**
Addresses https://github.com/rust-lang/infra-team/issues/103.

### CODE REVIEW

**Comment by geektype** [comment]
r? ghost

File: src/ci/docker/run.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ export MSYS_NO_PATHCONV=1
 script=`cd $(dirname $0) && pwd`/`basename $0`
 
 image=""
+
 dev=0
 
 while [[ $# -gt 0 ]]
```
Comment (matthiaskrgr) [inline src/ci/docker/run.sh:13]: ?

Comment (geektype) [reply src/ci/docker/run.sh:13]: Going to mess around with the CI so gonna use this PR to trigger builds. Let me know if there is a better way (this does feel hacky)

Comment (geektype) [reply src/ci/docker/run.sh:13]: Makes sense. I will close this PR in that case!

**Pull request closed without merging**